cdci_data_analysis/analysis/renku_helper.py

```diff
@@ -213,7 +213,7 @@ def generate_nb_hash(nb_obj):
 
     try:
         copied_notebook_obj['cells'][0].pop('id', None)
-        copied_notebook_obj['cells'].pop('id', None)
+        copied_notebook_obj['cells'][1].pop('id', None)
```

Comment:
you'd need to remove it from all cells, right? Right now it's two, but why not remove from all, in a loop?

Reply:
now loops over all the cells and pop the id out if present